docs/entities-intro.rst

```diff
@@ -4,14 +4,23 @@
 Introduction to Entities
 **************************
 
-If you've heard something about ODK Entities and want to better understand whether they're useful for your longitudinal data collection, you're in the right place! We've organized this page as a series of questions that are independent from each other so you can pick and choose the sections that interest you.
+If you've heard something about ODK Entities and want to better understand whether they're useful for your longitudinal data collection, you're in the right place! We've organized this page as a series of questions that are independent from each other so you focus on the topics that interest you.
 
 If you're someone who learns best by doing, you may prefer to jump straight into the tutorial on :doc:`building a community reporting tool with Entities <tutorial-community-reporting>` and to come back here if you have any questions. If you have a question that we haven't answered, you can `post on the forum <https://forum.getodk.org/c/support/6>`_.
 
+Overview:
+
+* :ref:`Concepts <entities-intro-concepts>`: how Entities fit with existing ODK concepts like forms and more general concepts like longitudinal data collection
```

Comment:
The summaries of the titles are repetitive and verbose. If you really think they are helpful, try these.

- Concepts: how Entities fit with ODK concepts like Forms and general concepts like longitudinal data collection
- Limitations: what Entities currently can't do and our plans for the future
- Alternatives: other concepts in ODK you can use, other software you may want to consider
- How to: mechanics of using Entities

Reply:
I think it's helpful to summarize the outcomes! It really helps orient the user and sets expectations. 

Here is a slight variation of what's above with what I had in my mind 

**Topics**

- **Overview:** how Entities fit with ODK concepts like Forms and longitudinal data collection
- **What's possible today**: current limitations and plans for the future
- **Alternative options**: other concepts in ODK you can use and other software to consider 
- **Get started:** How to use Entities in your workflow

Reply:
Thanks, I mostly took @alyblenkin's edits, just slightly changing "what's possible today" because we'd have to rewrite the section for that title to work.